sources/platform/actors/development/programming_interface/environment_variables.md

```diff
@@ -30,6 +30,7 @@ The Actor's process has several environment variables set to provide it with con
 | `ACTOR_DEFAULT_KEY_VALUE_STORE_ID` | ID of the key-value store where the Actor's input and output data are stored.                                                                                                                                    |
 | `ACTOR_DEFAULT_REQUEST_QUEUE_ID`   | ID of the request queue that stores and handles requests that you enqueue.                                                                                                                                            |
 | `ACTOR_INPUT_KEY`                  | The key of the record in the default key-value store that holds the Actor input. Typically it's **INPUT**, but it might be something else.                                                             |
+| `ACTOR_MAX_PAID_DATASET_ITEMS`     | If the Actor is paid per result, this will contain the limit set by the user on how much results they want to return. Do not return them more results, because they will be only charged for the limit they set and you might start accumulating loss.
```

Comment:
```suggestion
| `ACTOR_MAX_PAID_DATASET_ITEMS`     | If the Actor is paid per result, this will contain the limit set by the user on how many results they want to return. Do not return them more results because they will be only charged for the limit they set, and you might start accumulating loss.
```